packages/pytest-simcore/src/pytest_simcore/docker_api_proxy.py

```diff
@@ -22,7 +22,13 @@
 async def _wait_till_docker_api_proxy_is_responsive(
     settings: DockerApiProxysettings,
 ) -> None:
-    async with ClientSession(timeout=ClientTimeout(1, 1, 1, 1, 1)) as client:
+    async with ClientSession(
+        timeout=ClientTimeout(1, 1, 1, 1, 1),
```

Comment:
MINOR: i would add keyword arguments to know what these 1,1,1,1 refers to

Reply:
Yep, this is very unclear. what is 1? a second a millisecond? what is the point of defining all of them? since this is aiohttp, you can also define only 1 argument instead of having this super complicated thing. one of the timeout is getting a client from the pool, etc etc...
thought: Would it not make sense to create 1 client session for waiting instead of re-creating each time?

Reply:
managed to simplify this